config/spotbugs/exclude.xml

```diff
@@ -16,6 +16,10 @@
     <Method name="readCastVoteRecord"/>
     <Bug code="NP"/>
   </Match>
+  <Match>
+    <Class name="network.brightspots.rcv.RawContestConfig$CvrSource"/>
+    <Bug code="UwF"/>
+  </Match>
```

Comment:
What is this?

Reply:
spotbugs thinks `getUndervoteLabel` is unused, but it's used via reflection when jackson deserializes